CHANGELOG.md

```diff
@@ -89,6 +89,8 @@ All notable changes to this project will be documented in this file.
 * ### `niscope` (NI-SCOPE)
     * #### Added
     * #### Changed
+        * `GrpcStubInterpreter` now accepts the `meas_wfm_size` argument when calling `fetch_array_measurement`.
```

Comment:
We should write it in terms that are helpful to users.
`GrpcStubInterpreter` is an implementation detail and it has no value to users.

Can you re-write this in terms that are useful to users?

Reply:
```suggestion
        * When calling `fetch_array_measurement` in a MeasurementLink measurement plugin, the `meas_wfm_size` argument is now accepted.
```